ONSdigital/sdc-int-rh-ui
# 585 Fix prod config

# Motivation and Context

We want the app to default to safe config for production, and make it easier to understand how the config is set. The config template is now set by a `CONFIG_NAME` environment variable and will default to the `BaseConfig` if nothing is supplied, preventing the app from starting up with unexpected dev defaults.

Also, the "check" features of the info endpoint were broken and would fail semi silently, so I have removed the `check` parameter functionality entirely from that endpoint.

Also, while I was having to fix things across many files, I have made a start on replacing all the relatively imports with absolute.

# What has changed



- Stop the app defaulting to development/debug config
- Make the settings config clearer and more visible 
- Remove `check` parameter from the info endpoint and the unused supporting code
- Replace relative imports with absolute

# How to test?



Build locally and run in Docker dev, run the ATs, manually check the UI still appears the same.

# Links



https://trello.com/c/M2Z2vUC6/585-rh-ui-needs-to-use-prod-config-5
